src/org/opendatakit/briefcase/ui/reused/SourceConfigurationPanel.java

```diff
@@ -1,16 +1,22 @@
 package org.opendatakit.briefcase.ui.reused;
 
-import javax.swing.JPanel;
-
 public class SourceConfigurationPanel {
   private final SourceConfigurationPanelForm form;
+  private SourceDisplayPanelForm formSourceDisplay;
```

Comment:
To pull this off, I think we need a third "virtual" UI component. An empty `JPanel` would work, I think.

This JPanel is the container of this component and it will let you load **all the components** at once. Then you can make one of them visible, hide the rest, and toggle their visibility depending on the user's actions. 

We could create a `VirtualContainerForm` with just that `JPanel` and an API that would let us:
- `add(String name, JPanel view)`
- `navigateTo(String name)`

We could reuse this `VirtualContainerForm in other similar situations where we need to change what we're showing in a container

Reply:
This will be in the Controller or the View class?